include/translated-object.php

```diff
@@ -236,9 +239,29 @@ public function save_translations( $id, $translations ) {
 					}
 				}
 			}
+			return $translations;
+		} else {
+			return array();
 		}
 	}
 
+	/**
+	 * Returns translations after checking the translated post is in the right language
+	 *
+	 * @param array $translations Locales as keys and post ids as values.
+	 *
+	 * @return array
```

Comment:
Could you add `@since` and precise the array to `int[]` ? Also the keys are not locales but language codes.